app/controllers/resources_controller.rb

```diff
@@ -13,7 +13,7 @@ def new
   end
 
   def show
-    redirect_to(params.merge(:action => "edit"))
+    redirect_to :action => :edit
```

Comment:
Yeah you are right. I doesn't seem to cause any trouble. But just cross-checked and `resources#edit` doesn't make use of `params` anywhere. Not even in `before_filters`. So we can still remove it. But I wonder why codeclimate was complaining about it if they are not reference in edit anywhere...